src/components/ReactionView/hooks/useReaction.ts

```diff
@@ -38,7 +43,30 @@ const useReaction = (props: ReactionViewProps) => {
   const isSinglePress = showPopupType === GlobalConstants.onPress;
 
   const isLongPress = showPopupType === GlobalConstants.default;
+  const panResponder = useRef(
+    PanResponder.create({
+      onMoveShouldSetPanResponder: () => true,
+      onStartShouldSetPanResponder: () => true,
+      onPanResponderGrant: () => {},
+      onPanResponderMove: event => onGesture(event),
+      onPanResponderRelease: () => {
+        setPosition(0);
+      },
+    })
+  ).current;
 
+  const onGesture = async (event: GestureResponderEvent) => {
+    if (event.nativeEvent?.pageX <= 367) {
+      const currentItem = Math.floor(event.nativeEvent?.pageX);
+      if (currentItem) {
+        setPosition(currentItem);
```

Comment:
We can improve conditions via the ?? operator.
Like: setPosition(currentItem ?? 0);